modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionSamples/build.gradle

```diff
@@ -1,13 +1,26 @@
 task runGradleTest
 
 runGradleTest {
+	dependsOn ":sample-iframe:buildClientExtension"
 	dependsOn ":sampleThemeCSS:buildClientExtension"
 	dependsOn ":sampleThemeFavicon:buildClientExtension"
 
 	doLast {
+		Project sampleIFrameProject = project(":sample-iframe")
```

Comment:
can we move this file to sampleIframe to match the other directories?

Reply:
I need it to be named this specifically because it is testing a problem we have with projects with dashes in names.  However, I could move the other projects to also have dashes.

Reply:
ok, all projects having dashes is better. bchan doesn't like camel casing directories too